applications/serial_lte_modem/doc/Generic_AT_commands.rst

```diff
@@ -12,12 +12,12 @@ The following commands list contains generic AT commands.
 SLM version #XSLMVER
 ====================
 
-The ``#XSLMVER`` command requests the SLM version.
+The ``#XSLMVER`` command requests the versions of SDK in which the SLM application is built.
```

Comment:
```suggestion
The ``#XSLMVER`` command requests the version of the |NCS| in which the SLM application is built.
```
Should this mention the modem library as well?

Reply:
Added a line